Makefile

```diff
@@ -138,9 +138,19 @@ test: manifests generate fmt vet envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $$(go list ./... | grep -v /e2e) -coverprofile cover.out
 
 # Utilize Kind or modify the e2e tests to load the image locally, enabling compatibility with other vendors.
-.PHONY: test-e2e  # Run the e2e tests against a Kind k8s instance that is spun up.
+.PHONY: test-e2e  # Run the e2e tests against a Kind k8s instance that is spun up using emulator mode
```

Comment:
Why does this target depend on emulated mode?

Reply:
As of now, we have internally e2e tests running on emulator mode only, I added a variable here to show that the tests are running on emulator mode explicitly